docs/content/doc/administration/customizing-gitea.en-us.md

```diff
@@ -282,6 +282,22 @@ Place custom files in corresponding sub-folder under `custom/options`.
 
 To add custom .gitignore, add a file with existing [.gitignore rules](https://git-scm.com/docs/gitignore) in it to `$GITEA_CUSTOM/options/gitignore`
 
+## Customizing the git configuration
+
+Starting with Gitea 1.20, you can customize the git configuration via the `git.config` section.
+
+### Enabling signed git pushes
+
+To enable signed git pushes, set these two options:
+
+````
+[git.config]
+receive.advertisePushOptions = true
+receive.certNonceSeed = <randomstring>
+````
```

Comment:
```suggestion
```
[git.config]
receive.advertisePushOptions = true
receive.certNonceSeed = <randomstring>
```
```

Reply:
hm, not how it should look like... 😄 

Change the 4 ` to 3.

Reply:
Have edited it for you